ground/src/main/java/com/google/android/ground/ui/offlineareas/viewer/OfflineAreaViewerFragment.kt

```diff
@@ -34,13 +34,12 @@ import kotlinx.coroutines.launch
 @AndroidEntryPoint
 class OfflineAreaViewerFragment @Inject constructor() : AbstractMapContainerFragment() {
 
-  private lateinit var viewModel: OfflineAreaViewerViewModel
+  @Inject lateinit var viewModel: OfflineAreaViewerViewModel
```

Comment:
Why is this change required?

Reply:
When we can inject the viewModel, why to depend on creation?
Sort of a migration towards good practice

Reply:
Won't this create a new instance of viewModel whenever the fragment is destroyed and recreated? The existing code allows the same view model to be reused across lifecycle changes.

Reply:
> fragment is destroyed and recreated

Are we just talking about orientation change or recreation as a whole in a single app session?

Reply:
I'd recommend not doing this change in this PR since this is out of scope. Let's do it separately while ensuring there are no side-effects.

Reply:
ok,

Just tried both ways, new instance will always get created

Reply:
reverted this part